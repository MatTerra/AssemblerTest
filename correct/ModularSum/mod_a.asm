MOD_A: begin
section data
VAL: CONST 5
section text
Y: EXTERN
INPUT Y
LOAD VAL
ADD Y
STORE Y
JMPP MOD_B
MOD_B: EXTERN
L1: STOP
PUBLIC VAL
PUBLIC L1
EnD
