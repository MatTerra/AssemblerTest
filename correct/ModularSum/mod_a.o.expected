H: mod_a
H: 12
H: 010101010100
H: U mod_b 9
H: U y 1 5 7
H: D l1 10
H: D val 11
T: 12 00 10 11 01 00 11 00 07 10 14 5
